package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "custom-event-polyfill": "joscha/custom-event-polyfill#de24bd9",
     "debounce": "^1.0.0",
-    "weakmap": "0.0.6"
+    "webcomponents.js": "skatejs/webcomponentsjs#skate-v1"
```

Comment:
Is it just a fork of the webcomponents? Why can't we use normal version of webcomponents?